File: android/ui/components/UltraProTestimonial.kt
package com.ultrapro.ui.components

import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Card
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp

data class UltraProTestimonialItem(
    val quote: String,
    val author: String,
    val role: String
)

@Composable
fun UltraProTestimonial(
    item: UltraProTestimonialItem,
    modifier: Modifier = Modifier
) {
    Card(modifier = modifier) {
        Column(modifier = Modifier.padding(20.dp)) {
            Text(text = "“${item.quote}”", style = MaterialTheme.typography.bodyLarge)
            Text(
                text = item.author,
                style = MaterialTheme.typography.titleMedium,
                modifier = Modifier.padding(top = 12.dp)
            )
            Text(text = item.role, style = MaterialTheme.typography.bodyMedium)
        }
    }
}

File: lib/ui/components/ultra_pro_testimonial.dart
import 'package:flutter/material.dart';

class UltraProTestimonialItem {
  final String quote;
  final String author;
  final String role;

  const UltraProTestimonialItem({
    required this.quote,
    required this.author,
    required this.role,
  });
}

class UltraProTestimonial extends StatelessWidget {
  final UltraProTestimonialItem item;

  const UltraProTestimonial({
    super.key,
    required this.item,
  });

  @override
  Widget build(BuildContext context) {
    return Card(
      child: Padding(
        padding: const EdgeInsets.all(20),
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          children: [
            Text('“${item.quote}”', style: Theme.of(context).textTheme.bodyLarge),
            const SizedBox(height: 12),
            Text(item.author, style: Theme.of(context).textTheme.titleMedium),
            const SizedBox(height: 4),
            Text(item.role, style: Theme.of(context).textTheme.bodyMedium),
          ],
        ),
      ),
    );
  }
}
